pkgs/development/tools/misc/dura/default.nix

```diff
@@ -34,7 +40,6 @@ rustPlatform.buildRustPackage rec {
     '';
     homepage = "https://github.com/tkellogg/dura";
     license = licenses.asl20;
-    platforms = platforms.linux;
```

Comment:
Why removing this? How about adding `darwin` platform instead?

Reply:
by default `buildRustPackage` uses Rust's platforms, which includes darwin; and `dura` README mentions that it [runs even on windows](https://github.com/tkellogg/dura#does-this-work-on-my-os)
https://github.com/NixOS/nixpkgs/blob/f214afa5fb6452900e27776735db21f5092261b8/pkgs/build-support/rust/build-rust-package/default.nix#L156-L157

Reply:
Fair enough, thanks !